3rdparty/darknet/test/test_darknet.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Author: Yuki Furuta <furushchev@jsk.imi.i.u-tokyo.ac.jp>
+
+from __future__ import print_function
+import os
+import sys
+import subprocess
+import unittest
+import rospy
+import rospkg
+import time
+
+PKGNAME = "darknet"
+PKGPATH = rospkg.RosPack().get_path(PKGNAME)
+DATA_CFG_PATH = os.path.join(PKGPATH, "cfg", "coco.data")
+CFG_PATH = os.path.join(PKGPATH, "cfg", "yolo.cfg")
+WEIGHT_PATH = os.path.join(PKGPATH, "weights", "yolo.weights")
+IMG_PATH = os.path.join(PKGPATH, "data", "dog.jpg")
+
```

Comment:
Adding Two line space is better, you can check with linter: `rosrun roslint pep8 test_darknet.py`.